src/index.ts

```diff
@@ -1,7 +1,10 @@
 import Game from './core/Game';
 
+const fen = 'r1b1k1nr/p2p1pNp/n2B4/1p1NP2P/6P1/3P1Q2/P1P1K3/q5b1';
```

Comment:
Was it tested with extension of active color?

Reply:
Yes, it was - active color is optional and if not given the default value is Color.White